test/tor_test.go

```diff
@@ -18,7 +18,7 @@ func TestStartTor(t *testing.T) {
 		LocalPort:            "1111",
 		RemotePort:           "2222",
```

Comment:
i think you should also not add these changes and ignore them.

Reply:
done